ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -854,16 +854,41 @@ def osd_encryption_verification():
         log.info("Luks header info found for all the encrypted osds")
 
 
-def in_transit_encryption_verification():
+def ceph_config_dump():
+    """Get the 'ceph config dump' output.
+
+    Returns:
+        dict: The output of the 'ceph config dump' command as a dict.
     """
```

Comment:
```suggestion
    """
    Get the 'ceph config dump' output.

    Returns:
        dict: The output of the 'ceph config dump' command as a dict.
      
    """
```
Please apply to all places

Reply:
Formatted the doc strings